webapp/channels/src/i18n/en.json

```
@@ -2738,6 +2738,7 @@
   "adminConsole.list.table.rowsCount.20": "20",
   "adminConsole.list.table.rowsCount.50": "50",
   "adminConsole.list.table.rowsCount.show(rowsPerPage)": "rows per page",
+  "advanced_text_editor.remote_user_hour": "The time for {user} is: {time}",
```

```suggestion
  "advanced_text_editor.remote_user_hour": "The time for {user} is {time}",
```
I'd like to remove the colon

---

Done